src/tests/sqlsmith/src/sql_gen/relation.rs

```diff
@@ -71,11 +71,10 @@ impl<'a, R: Rng> SqlGenerator<'a, R> {
             for_system_time_as_of_proctime: false,
         };
         table.name = alias; // Rename the table.
-        let columns = table.get_qualified_columns();
-        (table_factor, columns, table)
+        (table_factor, table)
     }
 
-    fn gen_table_factor(&mut self) -> (TableFactor, Vec<Column>, Table) {
+    fn gen_table_factor(&mut self) -> (TableFactor, Table) {
```

Comment:
Removing `Vec<Column>` solves the problem in https://github.com/risingwavelabs/risingwave/pull/10343.

Because if we just clone schema, previously the convention is to make sure the schema column names were qualified.
But this is error prone. So I've changed that here to fix it.

At the call site we will require `get_qualified_columns` if it is required. You may ctrl-F `get_qualified_columns` to see this change in this PR.